src/v2/view-builder/views/signin/CustomButtons.js

```diff
@@ -0,0 +1,24 @@
+import { View, createButton } from 'okta';
+
+export default View.extend({
+  className: 'custom-buttons',
```

Comment:
should we use more specific 'custom-idp-buttons'

Reply:
these are custom button and not the idp buttons. They both are different.